spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/VectorizedSparkParquetReaders.java

```diff
@@ -49,4 +54,45 @@ public static ColumnarBatchReader buildReader(
                 expectedSchema, fileSchema, setArrowValidityVector,
                 idToConstant, ColumnarBatchReader::new));
   }
+
+  public static ColumnarBatchReader buildReader(
+          Schema expectedSchema,
```

Comment:
The formatting in this class and many other Arrow-related classes is off. However, we should at least match it. Iceberg uses 2 spaces for args. I think we are using 4 here.